test/e2e/crd_e2e_test.go

```diff
@@ -299,7 +299,7 @@ var _ = Describe("CRD Versions", func() {
 		// old CRD has been installed onto the cluster - now upgrade the subscription to point to the channel with the new CRD
 		// installing the new CSV should fail with a warning about data loss, since a storage version is missing in the new CRD
 		// use server-side apply to apply the update to the subscription point to the alpha channel
-		Eventually(Apply(subscription, func(s *operatorsv1alpha1.Subscription) error {
+		Eventually(ctx.Ctx().Apply(subscription, func(s *operatorsv1alpha1.Subscription) error {
```

Comment:
This is unnecessarily verbose. I think we can write a wrapper function that gives us back:

```go
Eventually(Apply(subscription, func(s *operatorsv1alpha1.Subscription) error {
```